config/initializers/devise.rb

```diff
@@ -13,7 +13,7 @@
   # ==> Mailer Configuration
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class with default "from" parameter.
-  config.mailer_sender = "please-change-me-at-config-initializers-devise@example.com"
+  config.mailer_sender = "confirm@railsgirlssummerofcode.org"
```

Comment:
hey @chashmeetsingh! thank you for this :) could you change this email address to `contact@rgsoc.org`? That's our current email address, so it would probably make the most sense. We can merge after that!